File: src/model/poe-api/Item.ts
export interface Item {
  id: string;
  name: string;
  category: any;
  typeLine: string;
  descrText: string;
  
  craftedMods: Array<string>;
  enchantMods: Array<string>;
  explicitMods: Array<string>;
  implicitMods: Array<string>;
  properties: Array<any>;
  additionalProperties: Array<any>;

  verified: boolean;
  x: number;
  y: number;
  icon: string;
  league: string;
  note: string;
}

File: src/go/model/poeapimodel/Item.go
package poeapimodel

// Item is exported
type Item struct {
	ID        string      `json:"id"`
	Name      string      `json:"name"`
	Category  interface{} `json:"category"`
	TypeLine  string      `json:"typeLine"`
	DescrText string      `json:"descrText"`

	CraftedMods          []string   `json:"craftedMods"`
	EnchantMods          []string   `json:"enchantMods"`
	ExplicitMods         []string   `json:"explicitMods"`
	ImplicitMods         []string   `json:"implicitMods"`
	Properties           []Property `json:"properties"`
	AdditionalProperties []Property `json:"additionalProperties"`

	X      uint   `json:"x"`
	Y      uint   `json:"y"`
	Icon   string `json:"icon"`
	League string `json:"league"`
}
